dashboard/src/pages/login/CoreConnect.tsx

```diff
@@ -112,22 +119,24 @@ const CoreConnect = () => {
         <h1 className="font-title text-h1 font-bold tracking-medium text-gray-300">
           Add a new core
         </h1>
-        {ShowBlurb && (
-          <WarningAlert>
-            <p>
-              You may need to adjust your network and browser settings. {' '}
-              <a
-                href="https://github.com/Lodestone-Team/lodestone/wiki/Known-Issues#network-errors"
-                target="_blank"
-                rel="noreferrer"
-                className="text-blue-200 underline hover:text-blue-300"
-              >
-              Learn more.
-              </a>
-            </p>
-          </WarningAlert>
+        {showBlurb && (
+          <div className='w-full'>
+            <WarningAlert>
```

Comment:
The `div` with the `className="w-full"` seems unnecessary, it'd be best if we just put the `classname` on the `WarningAlert`